release_notes.md

```diff
@@ -0,0 +1,38 @@
+ # Release notes
+
+- Woof Mobile APP
+
+All notable changes to this application will be documented in this file.
+
+## [Unreleased]
+
+`More information about working with the application will be added during development.`
+
+### User impacts
+
+Software requirements: 
+
+ * Deployment target for iOS SDKs - 15.0+
+ * Supported devices - iPhone only
+ * Supported modes - portrait
+ * Dark/night theme - deactivate
+ * The linters (Swiftformat and Swiftlint) integrated with Xcode in the build phase.
+
+### Notes
+
+Anyone can download and use the app features.
+
+### Support
```

Comment:
Why do we duplicating this info here? We had this in README.md already. The release notes should not describe the supported mode of the app. It should describe to users what capabilities are available in particular release.

Reply:
We decided  with the team to to delete this information. In previous review you wrote that you also don't mind. Now the file release_notes.md keeps only the title of the file and the app name, because we haven't made any releases yet.